pages/_document.js

```diff
@@ -44,6 +49,7 @@ class MyDocument extends Document {
           />
         </Head>
         <body>
+          <ColorModeScript initialColorMode="dark" />
```

Comment:
Here I have set the initial colormode script to dark, but if default mode is to be light then I'll swap this with light

Reply:
Previously, I hardcoded this to light because of the flickering. Now that v1 is support for the no flash script, I'd like to change this to use the system color mode setting.

Reply:
I think we have to import the no flash script too.

Reply:
[https://next.chakra-ui.com/docs/migration#colormode](https://next.chakra-ui.com/docs/migration#colormode) I followed from here to update the _document.js. So I think If we remove the initialColorMode here and enable `useSystemColorMode` to true then it would be better right?

Also there was no reference to this flash script you have mentioned, can I get some reference link to look over this again

Reply:
`<ColorModeScript />` referened in the link above 👍

Reply:
So we're good!